samanthas_telegram_bot/api_queries/smalltalk.py

```diff
@@ -0,0 +1,205 @@
+"""Functions for interaction with SmallTalk oral test service."""
+import asyncio
+import json
+import logging
+import os
+
+import httpx
+from dotenv import load_dotenv
+from telegram import Bot, Update
+from telegram.constants import ParseMode
+
+from samanthas_telegram_bot.conversation.auxil.log_and_report import log_and_report
+from samanthas_telegram_bot.data_structures.constants import ALL_LEVELS
+from samanthas_telegram_bot.data_structures.context_types import CUSTOM_CONTEXT_TYPES
+from samanthas_telegram_bot.data_structures.enums import SmalltalkTestStatus
+from samanthas_telegram_bot.data_structures.helper_classes import SmalltalkResult
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+
+URL_PREFIX = "https://app.smalltalk2.me/api/integration"
+
+HEADERS = {"Authorization": f"Bearer {os.environ.get('SMALLTALK_TOKEN')}"}
+TEST_ID = os.environ.get("SMALLTALK_TEST_ID")
+
+
+async def send_user_data_get_smalltalk_test(
+    first_name: str,
+    last_name: str,
+    email: str,
+    bot: Bot,
+) -> tuple[str | None, str | None]:
+    """Gets SmallTalk interview ID and test URL."""
+
+    async with httpx.AsyncClient() as client:
+        r = await client.post(
+            url=f"{URL_PREFIX}/send_test",
+            headers=HEADERS,
+            json={
+                "test_id": TEST_ID,
+                "first_name": first_name,
+                "last_name": last_name,
+                "email": email,
+            },  # TODO possibly webhook
+        )
+
+    logger.debug(f"Request headers: {r.request.headers}, request content: {r.request.content}")
+    data = json.loads(r.content)
+    logger.debug(f"Response: {r.status_code=}, {r.headers=}, {r.content=}, {data=}")
+
+    url = data.get("test_link", None)
```

Comment:
.get() default value for second arg is None, you are able not to write it :)

Reply:
TIL
Done